### DISCUSSION TITLE
Make a Stack for PWA

### DISCUSSION THREAD

**Discussion by JamesBrandts** [ShafSpecs/remix-pwa#122] [Ideas]
PWA is getting a lot of traction lately , but implementing some of the features on Remix, even using this tool, is no simple task. I think if there was a stack, somewhat basic like Indie, but with PWA features implemented from the start, It would be great.
Who doesn't want their app to work well on mobile, for it to be installable, to be able to send push notification.
Strap prisma and tailwind and I think it would be the thing.

**Comment by ShafSpecs** [comment]
This is in the works. I have plan to build a full-stack PWA template from next week. I am still not sure about the theme though, wether it should be retail (e-commerce), business, collaborative tool, etc. 

Open to ideas for this one

**Reply by JamesBrandts** [reply]
I understand and appreciate a lot the effort.
I think the simpler the better, because it'll be easier to implement and reach more use cases.
So each one can add the tools they need and build around the template.
At this point, after more than 3 days trying to show push notifications when my app is closed, if there was a template with only this part working, I would happily restart my project and build it around the template.
